data/darktableconfig.xml.in

```diff
@@ -2488,6 +2488,13 @@
     <shortdescription>arange buttons as bar below temp/tint sliders</shortdescription>
     <longdescription>if enabled, the buttons in white balance module will be aranged below temp/tint sliders, otherwise they'll be shown in a grid to the right of temp/tint sliders</longdescription>
   </dtconfig>
+  <dtconfig>
+    <name>plugins/darkroom/colorzones/bg_sat_factor</name>
+    <type min="0.1" max="1.0">float</type>
+    <default>0.5</default>
+    <shortdescription>saturation factor of the background</shortdescription>
```

Comment:
saturation factor of the color zone background

Reply:
fixed